fastkml/network_control.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (C) 2023  Christian Ledermann
+#
+# This library is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Union
+from fastkml.helpers import datetime_subelement
+from fastkml.helpers import datetime_subelement_kwarg
+from fastkml.helpers import subelement_text_kwarg
+from fastkml.helpers import text_subelement
+from fastkml.helpers import xml_subelement
+from fastkml.helpers import xml_subelement_kwarg
+from fastkml.registry import registry
+from fastkml.registry import RegistryItem
+from fastkml.base import _XMLObject
+from fastkml.times import KmlDateTime
+from fastkml.update import Update
+from fastkml.views import Camera, LookAt
+
+
+class _NetworkControl(_XMLObject):
+    """
+    Absrtact base class representing NetworkLinkControl in KML.
```

Comment:
Why do you need an abstract base class here? NetworkLinkControl is the one and only concrete Implementation.